src/components/Popup.js

```diff
@@ -32,8 +37,7 @@ export default class PopUp {
 
     this._popupElement.addEventListener("click", (evt) => {
       this._cardClickClose(evt);
-      }
-    );
-   
+      });
+    this._popupElement.addEventListener("keydown", (evt) => {this._handleEscClose});
```

Comment:
`    this._popupElement.addEventListener("keydown", (evt) => {this._handleEscClose});
`

Please, delete it

It doesn't work and you add the `Escape` handler in `open`